### ISSUE TITLE
My program fails

### ISSUE THREAD

**Issue by QiuShiHuang** [animetrics/PlistCpp#13]
I need to translate a Apple plist file.
I've included all downloaded files.
However, the program does not work properly.
This's what I've written in Main.

std::map dict;
Plist::readPlist("D:\111", dict);

This's where I was interrupted position.

In plist.cpp
throw Error("This type is not supported");

I think I need a hand to solve this problem.

**Comment by animetrics** [comment]
Please post the plist file and the code that you are using.

**Comment by QiuShiHuang** [comment]
How can I post?

**Comment by animetrics** [comment]
You can post the files using github gist https://gist.github.com and put the links here.

**Comment by QiuShiHuang** [comment]
I live in China, you know, some sites inaccessible. Unfortunately, including “gist.github.com”. I tried a few times and found that I was wrong. Because the referenced file is the wrong file. Iphone backup files are encrypted out, so I got it wrong. Sorry to trouble you. I'm not very good at English, I wrote most of the content is used by Google translation. If a syntax error, go sarcasms Google translation bar.   :)
